hydra-cluster/test/Test/DirectChainSpec.hs

```diff
@@ -187,13 +187,8 @@ spec = around showLogsOnFailure $ do
             postTx $ InitTx $ HeadParameters cperiod [alice]
             aliceChain `observesInTimeSatisfying` hasInitTxWith cperiod [alice]
 
-            someUTxOA <- generate $ genOneUTxOFor aliceCardanoVk
-            someUTxOB <- generate $ genOneUTxOFor aliceCardanoVk
-
-            postTx (CommitTx alice (someUTxOA <> someUTxOB))
-              `shouldThrow` (== MoreThanOneUTxOCommitted @Tx)
```

Comment:
should not we have a spec checking this is now valid?

Reply:
We have a valid commit transaction interaction just below on line 201/202